src/main/java/apaintus/controllers/AttributeController.java

```diff
@@ -27,6 +24,8 @@
     private Spinner<Double> shapeWidth;
     @FXML
     private Spinner<Double> shapeHeight;
+    @FXML
+    private TextArea shapeText;
```

Comment:
Est-ce que c'est vraiment un shapeText ou plus juste du text. On met pas du text sur des shapes

Reply:
changed to text